pkg/controller/seed-controller-manager/etcdrestore/etcd_restore_controller.go

```diff
@@ -144,6 +144,10 @@ func (r *Reconciler) Reconcile(ctx context.Context, request reconcile.Request) (
 		return reconcile.Result{}, err
 	}
 
+	if !cluster.Spec.Features[kubermaticv1.ClusterFeatureEtcdLauncher] {
+		return reconcile.Result{}, fmt.Errorf("etcdLauncher not enabled on cluster: %q", cluster.Name)
+	}
```

Comment:
Will this surface to users in any way? Are they gonna see it as failed in the UI? Maybe I'm wrong, but this is probably going to look like a backup forever in "pending", no?

Reply:
I was wondering if a valIdating webhook made sense here to prevent creating an `EtcdRestore` on the API, but I think we don't have a webhook for this in place yet?

Reply:
We already had a PR with a webhook :D. But then decided that a validating webhook would be a wrong approach as there is nothing unvalid with the EtcdRestore itself, rather something else(etcd-launcher is not enabled)

Reply:
But I agree that it would be good to inform the user in some way that the etcdrestore is not working. Not sure where to put this info? Maybe add new phase in EtcdRestore phases, `EtcdLauncherNotEnabled` ?

Reply:
Oh lol I was not aware of the webhook PR ... I would have approved it 😂 

But yeah, a phase is the next best thing IMHO. At least this should be moved into the private `reconcile` function so an event gets emitted.

Reply:
Good idea. Moved to private reconcile.

Reply:
Could I implement EtcdLauncherNotEnabled in a different story? This anyway is part of series of stories that came from https://github.com/kubermatic/kubermatic/issues/9345 issue. Probably should be put under an epic.